### PR TITLE
add note about stopImmediatePropagation()

### PR DESCRIPTION
**Pull request by mindplay-dk**
#### Motivation

Describes another small caveat.

### CODE REVIEW

File: files/en-us/web/api/event/stoppropagation/index.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ The **`stopPropagation()`** method of the {{domxref("Event")}}
 interface prevents further propagation of the current event in the capturing and
 bubbling phases. It does not, however, prevent any default behaviors from occurring; for
 instance, clicks on links are still processed. If you want to stop those behaviors, see
-the {{domxref("Event.preventDefault", "preventDefault()")}} method.
+the {{domxref("Event.preventDefault", "preventDefault()")}} method. It also does not
+prevent immediate propagation to other event-handlers - if you want to stop those,
```
Comment (jpmedley) [inline files/en-us/web/api/event/stoppropagation/index.md:23]:
```suggestion
prevent immediate propagation to other event-handlers. If you want to stop those,
```

File: files/en-us/web/api/event/stoppropagation/index.md [hunk 2]
```diff
@@ -19,7 +19,9 @@ The **`stopPropagation()`** method of the {{domxref("Event")}}
...
+see {{domxref("Event.stopImmediatePropagation", "stopImmediatePropagation()")}} method.
```
Comment (jpmedley) [inline files/en-us/web/api/event/stoppropagation/index.md:24]:
```suggestion
see {{domxref("Event.stopImmediatePropagation", "stopImmediatePropagation()")}.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Comment by mindplay-dk** [comment]
Ehm, I don't really know this syntax - I just copy/pasted some existing markup and modified the text... If it isn't correct, I don't really know how to correct it, sorry...

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]

**Review by jpmedley — CHANGES_REQUESTED** [review]
Apologies. I still left out a character when correcting the macro.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**